src/base32/alphabet.rs

```diff
@@ -70,7 +70,6 @@ make_base32_alpha!(
     b"110abcdefgh1jk1mn0pqrstvwxyz"
 );
 
-
```

Comment:
The small formatting changes in here are from leftover cargo fmt stuff I did not catch in the past (yes, this needs CI)